jumpcloud_bootstrap_template.sh

```diff
@@ -914,27 +1008,88 @@ if [[ ! -f $DEP_N_GATE_DONE ]]; then
     echo "Status: Creating System Account" >>"$DEP_N_LOG"
 
     ## Bind JumpCloud user to JumpCloud system
-    userBind=$(
-        curl -s \
-            -X 'POST' \
-            -A "${USER_AGENT}" \
-            -H 'Accept: application/json' \
-            -H 'Content-Type: application/json' \
-            -H 'x-api-key: '${APIKEY}'' \
-            -d '{ "attributes": { "sudo": { "enabled": '${admin}',"withoutPassword": false}}   , "op": "add", "type": "user","id": "'${userID}'"}' \
-            "https://console.jumpcloud.com/api/v2/systems/${systemID}/associations"
-    )
+    if [[ $SelfSignedRequest ]]; then
```

Comment:
TODO joe to remove, if API key is never used during search, the UserID isn't found

Reply:
Actually after looking into this, if a restart were to occur, this would still be helpful to self assign the USERID to the system w/o API Key